hack/update-codegen.sh

```diff
@@ -25,7 +25,7 @@ CODEGEN_PKG=$($GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
 cd $(dirname ${BASH_SOURCE[0]})/..
 
 source "${CODEGEN_PKG}/generate-groups.sh" \
-  "all" \
+  applyconfiguration,client,lister,informer \
```

Comment:
We must set proper commands since the behavior of `all` was changed.

https://github.com/kubernetes/code-generator/blob/791c213a776b4b92f9af0805c1cf57c118ba951c/generate-groups.sh#L60